make/CompileInterimLangtools.gmk

```diff
@@ -73,8 +86,10 @@ define SetupInterimModule
           $(TOPDIR)/src/$1/share/classes, \
       EXCLUDES := sun javax/tools/snippet-files, \
       EXCLUDE_FILES := $(TOPDIR)/src/$1/share/classes/module-info.java \
+          $(TOPDIR)/src/$1/share/classes/javax/tools/ToolProvider.java \
           Standard.java, \
-      EXTRA_FILES := $(BUILDTOOLS_OUTPUTDIR)/gensrc/$1.interim/module-info.java, \
+      EXTRA_FILES := $(BUILDTOOLS_OUTPUTDIR)/gensrc/$1.interim/module-info.java \
+                     $($1.interim_EXTRA_FILES), \
```

Comment:
Style nit. We try to avoid aligning broken up lines like this. Please just use 4 space continuation indent. (See https://openjdk.org/groups/build/doc/code-conventions.html)